server/questions.js

```diff
@@ -38,9 +38,12 @@ exports.addQuestionByType = function(qType, question, callback) {
 	questionToAdd.hint = question.hint;
 	questionToAdd.points = parseInt(question.points);
 	questionToAdd.visible = (question.visible === 'true');
-	questionToAdd.attempted = [];
-	questionToAdd.answered = [];
-	questionToAdd.attempts = [];
+	questionToAdd.correctAttempts = [];
+	questionToAdd.wrongAttempts = [];
+	questionToAdd.totalAttempts = [];
+	questionToAdd.correctAttemptsCount = 0;
+	questionToAdd.wrongAttemptsCount = 0;
+	questionToAdd.totalAttemptsCount = 0;
```

Comment:
this will need to be updated in common.js in the questionAttribute object

Reply:
Good catch I will update that tonight